integrations/wpseo/wpseo.php

```diff
@@ -483,4 +477,39 @@ public function translate_post_meta( $value, $key, $lang ) {
 
 		return pll_get_term( $value, $lang );
 	}
+
+	/**
+	 * Adds the yoast translatable metas to export.
+	 *
+	 * @param  array $metas An array of post metas (keyed with meta keys) to export.
+	 * @return array        A potentially modified array of post metas to export.
+	 *
+	 * @phpstan-param  array<string,int> $parsing_rules
+	 * @phpstan-return array<string,int> $parsing_rules
```

Comment:
I don' think your `@phpstan` param and return are correct. Thinks to subfields.
```suggestion
	 * @return array  The modified array of post metas to export.
```